src/udev/udev-builtin-net_id.c

```diff
@@ -1082,7 +1093,11 @@ static int get_link_info(sd_device *dev, LinkInfo *info) {
         if (r < 0 && r != -ENOENT)
                 return r;
 
-        (void) sd_device_get_sysattr_value(dev, "phys_port_name", &info->phys_port_name);
+        r = sd_device_get_sysattr_value(dev, "phys_port_name", &info->phys_port_name);
+        if (r >= 0 && sscanf(info->phys_port_name, "pf%uvf%u", &pf, &vf) == 2)
```

Comment:
you are not using `pf`, are you? hence drop it? and use `%*u` to skip it?

Reply:
OK